plugin/PlayAssetDelivery/src/test/java/com/google/play/core/godot/assetpacks/PlayAssetDeliveryTestHelper.java

```diff
@@ -46,4 +51,136 @@ public static Dictionary createAssetPackLocationsDictionary() {
     testDict.put("location2", innerDict2);
     return testDict;
   }
+
+  /**
+   * Creates a anonymous class, used for testing removePackSuccess(), that calls onSuccessListener
+   * the instant the listener is added.
```

Comment:
Although the listener callbacks are asynchronous, in our mock tasks we suppose the tasks will be completed instantly in order to save time and make test logic more streamlined.